Gateway/Request/CheckoutDataBuilder.php

```diff
@@ -211,9 +211,10 @@ protected function getOpenInvoiceData($order)
             $discountAmount += $item->getDiscountAmount();
 
             $priceExcludingTax = $item->getPriceInclTax() - $item->getTaxAmount();
+            $formattedPriceExcludingTax = $this->adyenHelper->formatAmount($priceExcludingTax, $currency);
 
             $formFields['lineItems'][] = [
-                'amountExcludingTax' => $priceExcludingTax,
+                'amountExcludingTax' => $formattedPriceExcludingTax,
                 'taxAmount' => $item->getTaxAmount(),
```

Comment:
just wandering if this does not need to be formatted as well ?

Reply:
good one - I will add it